### PR TITLE
Apply minor fixes across the project

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: sbt-akka-serialization-helper/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperKeys.scala [hunk 1]
```diff
@@ -20,6 +20,5 @@ trait AkkaSerializationHelperKeys {
 
   lazy val ashDumpPersistenceSchema = taskKey[File]("Dumps schema of classes that are persisted")
   lazy val ashDumpPersistenceSchemaOutputFile = settingKey[File]("Output file to dump persistence schema to")
-  lazy val ashDumpPersistenceSchemaOutputFilename = settingKey[String]("Filename to dump persistence schema to")
-  lazy val ashDumpPersistenceSchemaOutputDirectoryPath = settingKey[String]("Directory path to persistence schema dump")
+  lazy val ashDumpPersistenceSchemaOutputFilePath = settingKey[String]("Path of file to dump persistence schema to")
```
Comment (PawelLipski) [inline sbt-akka-serialization-helper/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperKeys.scala:23]: Feels more practical, at least judging by how it's currently used in Hydra

Comment (PawelLipski) [reply sbt-akka-serialization-helper/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperKeys.scala:23]: Or maybe not, lemme roll back

File: circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala [hunk 2]
```diff
@@ -15,7 +15,7 @@ import io.circe.jawn.JawnParser
  * An abstract class that is extended to create a custom serializer.
  *
  * After creating your subclass, don't forget to add your serializer and base trait to `application.conf`
- * (for more info [[https://doc.akka.io/docs/akka/2.5.32/serialization.html]])
+ * (for more info [[https://doc.akka.io/docs/akka/2.6.17/serialization.html]])
```
Comment (MarconZet) [inline circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala:18]:
```suggestion
 * (for more info [[https://doc.akka.io/docs/akka/current/serialization.html]])
```
found the right `latest` tag

**Review by MarconZet — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -55,24 +60,37 @@ Passing an object of class NOT extending SerializabilityTrait as a message may c
...
+file. It can be used for detecting accidental changes of events (journal) and states (snapshots) with a simple `diff`.
 
-To dump persistence schema, run:
+To dump persistence schema for each sbt module where `AkkaSerializationHelperPlugin` is enabled, run:
 
 ```shell
 sbt ashDumpPersistenceSchema
 ```
 
 Default file is `<sbt-module>/target/<sbt-module-name>-dump-persistence-schema-<version>.yaml` but it can be changed using sbt keys:
```
Comment (PawelLipski) [inline README.md:91]: To be reconsidered (maybe just single `target/dump-persistence-schema-<version>.yaml`)

Comment (MarconZet) [reply README.md:91]: It will be after #131

**Pull request merged**